src/kOS.Safe/Execution/CPU.cs

```diff
@@ -54,12 +57,6 @@ public int InstructionPointer
             set { currentContext.InstructionPointer = value; }
         }
         
-        /// <summary>
-        /// Where was the instruction pointer at the time this update
-        /// first started?
-        /// </summary>
-        int firstInstructionPointerInUpdate;
-
```

Comment:
this was only ever used as part of the old way to detect if we were still in trigger functions or if we'd popped back to mainline code, and it wasn't really working right anyway.